parm/obsforge_rocoto_template.xml.j2

```diff
@@ -88,4 +98,43 @@
 
 </task>
 
+<!--
+	============================================
+	Task: gfs_marine_dump
```

Comment:
why start with gfs and not gdas?

Reply:
no reason really, just a prototype for now. I'll add gdas the next time around.